vimeo-networking/src/main/java/com/vimeo/networking/utils/PrivacySettingsParams.java

```diff
@@ -0,0 +1,90 @@
+package com.vimeo.networking.utils;
+
+import com.vimeo.networking.Vimeo;
+import com.vimeo.networking.model.Privacy.CommentValue;
+import com.vimeo.networking.model.Privacy.EmbedValue;
+import com.vimeo.networking.model.Privacy.ViewValue;
```

Comment:
Import could just import `Privacy` for clarity, to force consumers to always use the qualified class name.